userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,124 @@
+
+
+#### Introductory Example
+
+In this example, the goal is to **localize** where in the image instances of the category **stop sign** are present. For this task we supply a single image of a stop sign as a starter image:
+
+
+```python
+import turicreate as tc
+
+# Load the starter images
+starter_images = tc.SFrame({'image':[tc.Image('stop_sign.png')],
+                   'label':['stop_sign']})
+
+# Load test images
+test_data = tc.SFrame({'image':[tc.Image('stop_sign_photo1.png'), 
+                                tc.Image('stop_sign_photo2.png')],
+                        'label':['stop_sign', 'stop_sign']})
+
+# Create a model. This step may take several hours                                      
+model = tc.one_shot_object_detector.create(starter_images, 'label')
+
+# Save predictions on the test set as a column of the test data SFrame
+test_data['predictions'] = model.predict(test_data)
+
+# Draw prediction bounding boxes on the test images
+test_data['image_with_predictions'] = \
+	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions']) 
+
+# Evaluate the model and save the results in a dictionary
+metrics = model.evaluate(test_data)
+
+# Save the model for later use in TuriCreate
+model.save('mymodel.model')
+
+# Export for use in Core ML
+model.export_coreml('MyCustomOneShotDetector.mlmodel')
+```
+
+Previewing the synthetic images generated for training is useful to gauge the kind of test images the model would be trained to handle. We can preview the augmented images generated and used by the model for training as follows:  
+
+
+```python
+augmented_images = \
+	tc.one_shot_object_detector.util.preview_synthetic_training_data(starter_images, 'label')
+
+augmented_images.explore()
+```
+
+![Explore augmented images](images/augmented_images_explore.jpg)
+
+Some examples of synthetic training data generated:
+
+![Sample augmented images](images/augmented_images_collage.jpg)
+
+*Note: In order to view the exact augmented images used to train the model, the same seed should be used which was used to train the model.*
+
+
+To visualize the predictions made on the test set:
+
+
+```python
+test_data['image_with_predictions'] = \
+	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions'])
+
+test_data.explore()
+```
+
+![Prediction test data](images/test_data_explore.jpg)
+
+Examples of test image predictions:
+
+![Sample prediction image - 1 and 2](images/sample_prediction_images_1_2.jpg)
+
+
+#### How it works
+
+To begin training for one-shot object detector, a set of synthetic training data needs to be generated using the provided starter training image(s). These images are augmented by applying yaw, pitch, and roll rotations. These operations are applied on a set of background images. The background images may be provided by the user. If no background images are provided, a set of default background images are automatically downloaded and used.
```

Comment:
How about we condense this as follows:

"To begin training a one-shot object detector, a set of synthetic training data needs to be generated using the provided starter training image(s). **The synthetic images are generated by applying yaw, pitch, and roll rotations on a set of background images.** The background images may be provided by the user. If no background images are provided, a set of default background images are automatically downloaded and used."

Reply:
I like this edit.